src/wp-includes/media.php

```diff
@@ -1843,6 +1845,11 @@ function wp_filter_content_tags( $content, $context = null ) {
 				$filtered_image = wp_img_tag_add_loading_attr( $filtered_image, $context );
 			}
 
+			// Add 'decoding=async' attribute unless a 'decoding' attribute is already present.
+			if ( false === strpos( $filtered_image, ' decoding=' ) ) {
```

Comment:
For readability, I'd suggest switching this to `str_contains()`. This function is available in PHP 8 and a polyfill introduced in WP 5.9 making it available for use on all of Core's supported PHP versions.

```suggestion
			if ( ! str_contains( $filtered_image, ' decoding=' ) ) {
```

References:

* https://www.php.net/manual/en/function.str-contains.php
* https://core.trac.wordpress.org/changeset/52039

Reply:
Nice!

Reply:
I think this check should move to `wp_img_tag_add_decoding_async_attr()` as a guard clause. See my note here https://github.com/WordPress/wordpress-develop/pull/2409/files#r837622678.